dockstore-webservice/src/test/java/io/dockstore/webservice/helpers/ZenodoHelperTest.java

```diff
@@ -44,4 +47,58 @@ public void extractDoiFromBadDoiUrl() {
         Assert.assertNotEquals("10.5072/zenodo.372767", doi);
     }
 
+    @Test
```

Comment:
These are good, but you should also have unit tests for AliasResourceInterface.checkAlias directly, in a new class AliasResourceInterfaceTest

Reply:
done